src/bin/init.rs

```diff
@@ -20,9 +23,11 @@ pub fn make_subcommand<'a, 'b>() -> App<'a, 'b> {
 pub fn execute(args: &ArgMatches) -> Result<()> {
     let book_dir = get_book_dir(args);
     let mut builder = MDBook::init(&book_dir);
+    let mut config = config::Config::default();
 
     // If flag `--theme` is present, copy theme to src
     if args.is_present("theme") {
+        config.set("output.html.theme", "src/theme")?;
```

Comment:
I think this actually conflicts with the HTML config's `theme_dir()` function. By default that will take the book root and append `theme` to it (i.e. you get `/path/to/book/theme`) whereas this will put the themes directory in `src/` (i.e. you get `/path/to/book/src/theme`)... Was that intentional?

I know the user guide says the `theme` directory should be at the book root:

> The theme is totally customizable, you can selectively replace every file from the theme by your own by adding a theme directory next to src folder in your project root.

Reply:
So the reason I chose this one was because I ran `cargo run -- init --theme test_book` to determine where the theme was initially copied to when this argument was given and I found that it was copied under `mdBook/test_book/src/theme`. Unfortunately I haven't fully wrapped my head around this so I am not sure if I have used this correctly... Is that where it should be copied to initially? More than happy to change it to "theme" if that is the correct manner. 👍